airbyte-bootloader/src/main/java/io/airbyte/bootloader/BootloaderApp.java

```diff
@@ -46,14 +46,38 @@
   private static final AirbyteVersion VERSION_BREAK = new AirbyteVersion("0.32.0-alpha");
 
   private final Configs configs;
+  private Runnable postLoadExecution;
 
   @VisibleForTesting
   public BootloaderApp(Configs configs) {
     this.configs = configs;
   }
 
+  /**
+   * This method is exposed for Airbyte Cloud consumption.
+   *
+   * @param configs
+   * @param postLoadExecution
+   */
+  public BootloaderApp(Configs configs, Runnable postLoadExecution) {
```

Comment:
In Cloud we would extend this by creating a wrapped bootloader app and only write the seed spec that we want to apply. In this manner we would stop writing seed specs we don't want to exist in Cloud (like we do today).